src/PKSim.Presentation/Services/EditParameterPresenterTask.cs

```diff
@@ -53,7 +53,7 @@ public void EditTableFor(ICommandCollector presenter, IParameterDTO parameterDTO
             if (!tablePresenter.Edit(parameter))
                return;
 
-            presenter.AddCommand(_parameterTask.SetParameterFomula(parameter, tablePresenter.EditedFormula));
+            presenter.AddCommand(_parameterTask.UpdateTableFormula(parameter, tablePresenter.EditedFormula));
```

Comment:
Same here. This was really the issue. When the formula was edited, BOTH parameter in simulation and building block parmaeter in simulation would get the same reference to the formula, leading to two formulas with different ids after deserializing